addons/project_sms/__manifest__.py

```diff
@@ -1,11 +1,10 @@
-# -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
 {
-    'name': "Project - SMS",
-    'summary': 'Send text messages when project/task stage move',
-    'description': "Send text messages when project/task stage move",
-    'category': 'Hidden',
+    'name': 'Task Stage Notification via SMS',
+    'summary': 'Automatically send an SMS text message to your customers when a task reaches a specific stage of the project.',
+    'description': "Automatically send an SMS text message to your customers when a task reaches a specific stage of the project.",
```

Comment:
imo, you don't need "text message" after "SMS", because SMS's are always text messages